### ISSUE TITLE
Unified Inbox displayed when selecting multiple notifications from same account

### ISSUE THREAD

**Issue by garrettboone** [thunderbird/thunderbird-android#6034]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.000

### Where did you get the app from?

F-Droid

### Android version

LineageOS microg build 20220405 (lake)

### Device model

Moto G7

### Steps to reproduce

1. Show Unified Inbox
2. Select account B inbox (now showing only B messages)
3. Write two messages to account A
4. dismiss app, go to home screen
5. pull down notification and select two messages for account A

### Expected behavior

Show inbox of account A

### Actual behavior

Unified Inbox is shown instead of inbox from account A

Additional troubleshooting notes:
- Selecting an individual message from notification opens that message ok
- I decided to put Unified Inbox in the title, because I also have a behavior where I am in an account inbox and then read a message, and delete it - and it returns me to Unified Inbox rather than the account inbox.

### Logs

[k9-log.txt](https://github.com/k9mail/k-9/files/8549685/k9-log.txt)

Around 8:19 notifications were selected

**Comment by cketti** [comment]
This is the intended behavior now. If all messages with notifications belong to the Unified Inbox, the Unified Inbox will be opened.

> I decided to put Unified Inbox in the title, because I also have a behavior where I am in an account inbox and then read a message, and delete it - and it returns me to Unified Inbox rather than the account inbox.

This sounds like a bug. Please open a new issue and include detailed instructions on how to reproduce it.

**Comment by garrettboone** [comment]
OK good to know, I will get new logs together for the other one. Thank you.

**Comment by garrettboone** [comment]
@cketti I can do a PR for this also if I understand you correctly: https://github.com/k9mail/k9mail.github.io/blob/bf3a85d2930cff463dab088872141c5df7b0c874/_documentation/notifications.md?plain=1#L31

`Tapping on the stacked notification clears it and opens the Inbox for that account.`

**Comment by cketti** [comment]
We'll hopefully switch to the documentation in a separate repository soon. Please update this file instead: https://github.com/k9mail/k9mail-docs/blob/1ed793dc41451db49c255135ca4b8d014a904c6d/docs/en/current/reading/notifications.md

**Comment by garrettboone** [comment]
@cketti Ok I figured out the other sequence that I thought was a bug. It's a little different than I described. But is it a bug?
1. There are multiple messages from multiple accounts in notifications.
2. In app, Unified Inbox is enabled
3. select one message from a stacked notification group, and it goes to that message
4. Delete that message
5. It shows Unified Inbox rather than account inbox (bug?) 
6. change to the expected account inbox
7. remaining stacked notifications from that account are still showing (bug?) 
8. the individual notification dropdowns in the stack for that account are now missing and only the entire notification stack for that account can be selected (bug?)
